### PR TITLE
Move plugins to the new organization

### PR DESCRIPTION
**Pull request by alepuccetti**
This PR removes the plugins from scope repository because now they are hosted in a new organization https://github.com/weaveworks-plugins

Update the documentation to be more generic and to point to the new repositories.

Dependencies:
- [x] https://github.com/weaveworks-plugins/scope-iowait/pull/1
- [x] https://github.com/weaveworks-plugins/scope-http-statistics/pull/1
- [x] https://github.com/weaveworks-plugins/scope-http-statistics/pull/2

@alban

### CODE REVIEW

**Comment by 2opremio** [comment]
@alepuccetti Mind rebasing?

**Comment by alepuccetti** [comment]
Done.
Also, I am making an other PR on `weaveworks-plugins/scope-http-statistic` today. This should be merged after that one too, because this PR lists in the `HTTP statistics` a new metrics (HTTP responses codes) which will be added to the plugin later today.

**Comment by alepuccetti** [comment]
Relevant to https://github.com/weaveworks/scope/issues/1963

File: examples/plugins/README.md [hunk 1]
```diff
@@ -51,19 +42,18 @@ scan that directory every 5 seconds, to look for sockets being added
...
 There are several interfaces a plugin may (or must) implement. Usually
 implementing an interface means handling specific requests. These
 requests are described below.
 
-### Reporter interface
+#### Reporter interface
 
-Plugins _must_ implement the reporter interface. Implementing this
-interface means listening for HTTP requests at `/report`.
+Plugins _must_ implement the reporter interface because Scope uses it to discover which other interfaces the plugin implements and what data and controls the plugin exposes by doing `GET /report`.
```
Comment (2opremio) [inline examples/plugins/README.md]: Please try to simplify the phrasing

File: examples/plugins/README.md [hunk 2]
```diff
@@ -1,39 +1,30 @@
...
-If the running plugin was picked up by Scope, you will see it in the
-list of `PLUGINS` in the bottom right of the UI.
-
-## Plugin ID
+
+## Official Plugins
+
+You can find all the official plugins at [Weaveworks Plugins](https://github.com/weaveworks-plugins).
+
+* [IOWait](https://github.com/weaveworks-plugins/scope-iowait): a Go plugin using [iostat](https://en.wikipedia.org/wiki/Iostat) to provide host-level CPU IO wait or idle metrics.
```
Comment (2opremio) [inline examples/plugins/README.md]: I am not sure it's a good idea to have this here. We probably want it under `/site`. I am OK with it living here for now but maybe the listing (ant the rest of the documentation) should live on `/site` and we can link it from here.

@abuehrle thoughts?

**Comment by 2opremio** [comment]
Minor comments. Otherwise LGTM.

Sorry it took so long to review @alepuccetti :S

File: examples/plugins/README.md (hunk 2 above)
Comment (abuehrle) [inline examples/plugins/README.md]: It's fine as a readme for now. I will update the website docs with this at a later time.

Comment (2opremio) [inline examples/plugins/README.md]: Cool!

Comment (alepuccetti) [inline examples/plugins/README.md]: It is very much working in progress

**Comment by 2opremio** [comment]
@alepuccetti, I just realized that before merging this we should also fix the links under site (see https://www.weave.works/documentation/scope-latest-plugins/ , for instance https://github.com/weaveworks/scope/tree/master/examples/plugins/http-requests will be broken ) and redeploy the documentation.

**Comment by 2opremio** [comment]
Also, if you don't mind, please rebase again since there are some JS tests breaking (which I believe are fixed in master)

**Comment by alepuccetti** [comment]
> Also, if you don't mind, please rebase again since there are some JS tests breaking (which I believe are fixed in master)
 
My branch is up to date. I will trigger a rebuild

**Comment by 2opremio** [comment]
I will merge this and republish the docs to avoid broken links.

**Comment by 2opremio** [comment]
Done: https://www.weave.works/documentation/scope-latest-plugins/

**Pull request merged**